src/bundle/partial.ts

```diff
@@ -27,49 +43,65 @@ export async function prepareBundlePartialFiles(path: string, snag: LogSnag, org
   return manifest
 }
 
-export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: Options, appId: string, name: string) {
-  const spinner = spinnerC()
-  spinner.start('Preparing partial update')
-  const uploadResponse: uploadUrlsType[] = await manifestUploadUrls(apikey, appId, name, manifest)
+export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: any, config: CapacitorConfig, appId: string, name: string) {
+  try {
+    const baseUrl = config?.plugins?.CapacitorUpdater?.cloudflareBaseUrl ?? defaultApiHost
```

Comment:
Rename cloudflareBaseUrl to something more generic like partialUploadUrl, and more specific to the endpoint